pyproject.toml

```diff
@@ -39,7 +39,7 @@ hdf_reader = [
     "numcodecs"
 ]
 icechunk = [
-    "icechunk==0.1.0a8",
+    "icechunk>=0.1.0a12",
```

Comment:
@jsignell what's the motivation for using a12 rather than a15?

Reply:
I saw this comment: https://github.com/zarr-developers/VirtualiZarr/pull/378#issuecomment-2608168675 and so I thought a12 would be ok. And I looked at the codebase for the latest tagged version of icechunk and I saw [this line](https://github.com/earth-mover/icechunk/blob/a13459b118e2d7eb0774f398064bdab34997b5c2/icechunk-python/pyproject.toml#L23)  which made me nervous about using newer than a12

Reply:
nice, thanks for sharing!